lib/awful/keygrabber.lua

```diff
@@ -544,6 +544,21 @@ function keygrabber:set_allowed_keys(keys)
     end
 end
 
+--TODO v5 remove this
+function keygrabber:set_release_event(event)
+    gdebug.deprecate("release_event has been renamed to stop_event",
+        -- It has never been in a release, so it can be deprecated right away
+        {deprecated_in=4}
+    )
```

Comment:
Just remove the second argument to this function. That way, `gears.debug.deprecate` does not do any version check.